src/test/java/uk/gov/pay/api/validation/PaymentRequestValidatorTest.java

```diff
@@ -49,7 +50,10 @@ public void validParameters_withWelshLanguage_shouldSuccessValue() {
     @Test
     public void validateUnsupportedLanguage_shouldFailValue() {
         CreatePaymentRequest createPaymentRequest = createPaymentRequestBuilderWithReturnUrl().language("unsupported language").build();
+
         expectedException.expect(PaymentValidationException.class);
+        expectedException.expect(aValidationExceptionContaining("P0102", "Invalid attribute value: language. Must be \"en\" or \"cy\""));
```

Comment:
Do we really need both `expectedException.expect(…)` lines?

Reply:
Hmmm partly I wanted to minimise the scope of the refactoring to aid review and minimise chances of introducing an error. However, looking at `aValidExceptionContaining` its a type safe matcher based upon `PaymentValidationException`, so you're right, we don't! I'll remove.